frontend/bulk-export/bulk-export-modal.component.tsx

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import Modal from "../util/modal.component";
+import { CsvDataService } from "../util/export-to-csv.component";
+
+// Hard-coded data in the format exportToCsv is expecting
+const testData = [
+  {
+    firstName: "Celica",
+    lastName: "McMichel",
+    phone: "8012220394",
+    address: "147 Main street, Salt Lake City, UT 32231",
+  },
+  {
+    firstName: "John",
+    lastName: "Paul",
+    phone: "8290020302",
+    address: "147 Main street, Salt Lake City, UT 32231",
+  },
+  {
+    firstName: "Jack",
+    lastName: "Johnson",
+    phone: "9203929303",
+    address: "147 Main street, Salt Lake City, UT 32231",
+  },
+];
+
+export default function BulkExportModal(props: BulkExportModalProps) {
```

Comment:
This modal will need to handle when all clients are selected across all pages. Currently only one page of clients is loaded at a time, but for exporting all at once you'd need to go through and download each page.